woocommerce/woocommerce
# [Experimental] Filter blocks: Add block preview

### Changes proposed in this Pull Request:

This PR adds the block preview to each Product Filter block.

Closes #44748.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Navigate to Single Product template (Appearance > Editor > Templates > Product Catalog template - or any archive template).
2. Click on "Edit" to open the WordPress editor.
3. In the editor, you should see a block inserter icon or an option to "Add Block". Click on it.
4. In the block inserter, search for "Product Filter" or browse the available blocks until you find them.
5. For each Product Filter [...] (Beta) that appears, hover over it and make sure the block preview is displayed.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Add block preview to filter blocks

#### Comment 

</details>
